src/main/webapp/data/userMapData.json

```diff
@@ -702,9 +731,11 @@
       "TIAS Business School",
       "TU Delft",
       "University of Amsterdam",
+      "University of Applied Sciences Leiden",
       "University of Groningen",
       "University of Inholland",
       "University of Twente",
+      "Universiteit van Amsterdam",
```

Comment:
Put this line in front of `University of Amsterdam`. BTW do they refer to the same school?

Reply:
yes, they do refer to same university, so i m deleting this one "Universiteit van Amsterdam".